ghcide/test/data/plugin-recorddot/RecordDot.hs

```diff
@@ -1,6 +0,0 @@
-{-# LANGUAGE DuplicateRecordFields, TypeApplications, TypeFamilies, UndecidableInstances, FlexibleContexts, DataKinds, MultiParamTypeClasses, TypeSynonymInstances, FlexibleInstances #-}
```

Comment:
Removing this because one of the tests being removed says
`ignoreForGHC92Plus "No need for this plugin anymore!"`